packages/stable/src/api/geospatial/types.ts

```diff
@@ -269,3 +269,38 @@ export interface GeospatialCRSResponse
 export interface GeospatialSridId {
   srid: number;
 }
+
+export type GeospatialComputeFunction =
+  | {
+      stTransform: GeospatialGeometryTransformComputeFunction;
+    };
+
+export interface GeospatialGeometryTransformComputeFunction {
+  geospatialGeometryComputeFunction: GeospatialGeometryComputeFunction;
+  srid: number;
+}
+
+export type GeospatialGeometryComputeFunction =
+  | {
+      geometryValue: GeospatialGeometryValueComputeFunction;
+    };
+
+export interface GeospatialGeometryValueComputeFunction {
```

Comment:
```suggestion
export interface GeospatialExtendedWellKnownText {
```

Reply:
should replace `GeospatialGemetryValueComputeFunction` in above functions as well.